src/codecs/avif/encoder-meta.ts

```diff
@@ -12,12 +14,14 @@ export const label = 'AVIF';
 export const mimeType = 'image/avif';
 export const extension = 'avif';
 export const defaultOptions: EncodeOptions = {
-  minQuantizer: 16,
-  maxQuantizer: 16,
+  minQuantizer: 33,
+  maxQuantizer: 63,
```

Comment:
I invented these defaults by looking for numbers that seem to match the quality of the default mozjpeg options.

Reply:
Actually, that `maxQuantizer` looks pretty ugly on the illustration example.

Reply:
I think you're supposed to set them both to the same value. min/max makes sense only for video with variable rate encoding.

Reply:
I thought the same - I'd only ever seen it in target bitrate encoding, however it seems to do something here. https://5f4512e9cdb7030008747032--squoosh.netlify.app/ - try the "artwork" example image. The low min setting causes a lot of smoothing. It made me wonder if min/max was similar to SNS in WebP. I'll do some research though.